src/nowplaying.html

```diff
@@ -163,20 +177,17 @@ <h2 class="nowPlayingPageTitle"></h2>
         </div>
         <div class="playlistSection">
             <div class="playlistSectionButton flex align-items-center justify-content-center">
```

Comment:
For TV. Keep focus inside container with horizontal keyboard navigation.
```suggestion
            <div class="playlistSectionButton flex align-items-center justify-content-center focuscontainer-x">
```